sys/net/gnrc/conn/udp/gnrc_conn_udp.c

```diff
@@ -14,6 +14,7 @@
  */
 
 #include <errno.h>
+#include "assert.h"
```

Comment:
it refers to [assert.h](https://github.com/RIOT-OS/RIOT/blob/master/core/include/assert.h).